### PR TITLE
Product page

### PR DESCRIPTION
**Pull request by VedanthB**
- added the product page
- this is the preview link https://deploy-preview-2--romantic-mccarthy-7219f3.netlify.app/pages/product-page/product-listing.html

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -14,6 +14,13 @@
 .bg-regal-blue-dark {
   background-color: #000022;
 }
+.text-regal-blue {
+  color: #2a2840;
+}
+
+.text-regal-blue-dark {
+  color: #000022;
+}
```
Comment (Richa-rani02) [inline index.css]: suggestion: define variable name for colors

**Review by Richa-rani02 — CHANGES_REQUESTED** [review]

File: pages/product-page/product-listing.css [hunk 2]
```diff
@@ -0,0 +1,15 @@
+.divider {
+  border: 1px solid var(--regal-blue-dark);
+  width: 80%;
+  margin: auto;
+}
+.filters-sidebar-container {
+  height: 100%;
+  width: 20%;
+  /* border: 1px solid var(--regal-blue-dark); */
+  border-radius: 0.5rem;
```
Comment (Hardiegogo) [inline pages/product-page/product-listing.css:10]: for such a small value, you can use px.

**Review by Hardiegogo — COMMENTED** [review]
Good work Vedanth.
I have suggested a small change below.

File: pages/product-page/product-listing.css (hunk 2 above)
Comment (VedanthB) [reply pages/product-page/product-listing.css:10]: okay , will make the changes

File: index.css (hunk 1 above)
Comment (VedanthB) [reply index.css]: will make the changes

**Pull request merged**